db/migrate/20230731090232_add_fields_to_accounts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddFieldsToAccounts < ActiveRecord::Migration[6.1]
+  def change
+    change_table :accounts, bulk: true do |t|
+      t.string :domain
+      t.string :organization
+      t.jsonb  :services, default: []
```

Comment:
Én egész keveset fejlesztettem postgres alatt, viszont a fenti gyanús.
"`Services` (required - would be nice to store it as jsonb list)" => Ez nekem azt jelentené, hogy:
1. Nincs default értéke a `:services`-nek, mert ezt nem specifikálta az issue.
2. Nem lehet üres a `:services`, tehát adatbázis oldalról nem szabadna ide engednünk `null` vagy `[]` vagy bármilyen "üres" jsonb értéket.